### ISSUE TITLE
How to train?

### ISSUE THREAD

**Issue by wzg722** [arvindrajan92/DTrOCR#23]
How to train?

**Comment by DemissieDaniel** [comment]
@arvindrajan92 how to train?

**Comment by arvindrajan92** [comment]
> @arvindrajan92 how to train?
> 
> 

Hi @DemissieDaniel 

Would this notebook be of use: https://github.com/arvindrajan92/DTrOCR/blob/main/examples/IAMFineTune.ipynb.

You would need to build the dataset using the resources outlined in the original DTrOCR paper. 

Hope this helps.

**Comment by PanFei748** [comment]
> > [@arvindrajan92](https://github.com/arvindrajan92) how to train?
> 
> Hi [@DemissieDaniel](https://github.com/DemissieDaniel)
> 
> Would this notebook be of use: https://github.com/arvindrajan92/DTrOCR/blob/main/examples/IAMFineTune.ipynb.
> 
> You would need to build the dataset using the resources outlined in the original DTrOCR paper.
> 
> Hope this helps.

Hi，why does it perform well on the training set, but very poorly on the validation set, with no decrease in loss

**Comment by arvindrajan92** [comment]
> > > [@arvindrajan92](https://github.com/arvindrajan92) how to train?
> > 
> > 
> > Hi [@DemissieDaniel](https://github.com/DemissieDaniel)
> > Would this notebook be of use: https://github.com/arvindrajan92/DTrOCR/blob/main/examples/IAMFineTune.ipynb.
> > You would need to build the dataset using the resources outlined in the original DTrOCR paper.
> > Hope this helps.
> 
> Hi，why does it perform well on the training set, but very poorly on the validation set, with no decrease in loss

@PanFei748 that would be because the model is not a pre-trained one (like it is mentioned in the paper). I am still working on it on my spare time therefore the progress is slow. Hope I can get a version of pre-trained weights across soon.

**Comment by PanFei748** [comment]
> > > > [@arvindrajan92](https://github.com/arvindrajan92) how to train?
> > > 
> > > 
> > > Hi [@DemissieDaniel](https://github.com/DemissieDaniel)
> > > Would this notebook be of use: https://github.com/arvindrajan92/DTrOCR/blob/main/examples/IAMFineTune.ipynb.
> > > You would need to build the dataset using the resources outlined in the original DTrOCR paper.
> > > Hope this helps.
> > 
> > 
> > Hi，why does it perform well on the training set, but very poorly on the validation set, with no decrease in loss
> 
> [@PanFei748](https://github.com/PanFei748) that would be because the model is not a pre-trained one (like it is mentioned in the paper). I am still working on it on my spare time therefore the progress is slow. Hope I can get a version of pre-trained weights across soon.

First of all, thank you very much for your work. The loss of train data < 0.04 and the accuracy > 98.9, but in val data, the loss remains around 2.7 and the accuracy is around 0.7, thank you for your work again.

**Comment by ducanh841988** [comment]
@arvindrajan92 could you double check on the image size? I saw you use very small size of image.
